src/esp/metadata/managers/StageAttributesManager.cpp

```diff
@@ -323,8 +330,9 @@ void StageAttributesManager::setValsFromJSONDoc(
   std::string semanticFName = stageAttributes->getSemanticAssetHandle();
   semanticFName = this->setJSONAssetHandleAndType(
       stageAttributes, jsonConfig, "semantic_asset_type", "semantic_asset",
-      semanticFName,
-      std::bind(&StageAttributes::setSemanticAssetType, stageAttributes, _1));
+      semanticFName, [stageAttributes](auto&& PH1) {
+        stageAttributes->setSemanticAssetType(std::forward<decltype(PH1)>(PH1));
```

Comment:
What's the "PH1"? If anything, doesn't look like it matches our coding conventions :)

Reply:
These fixes were automatically generated by clang-tidy. It's short for std::PlaceHolder::_1, since the std::binds didn't not have any informative args before. @jturner65 Another advantage of lambdas is that you can name your place holders to be more informative.

Reply:
Ah, autogenerated, I see :)

Reply:
Very nice, tyvm :)